styles.css

```
@@ -1 +1,66 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+.container {
+        display: grid;
+        grid-auto-columns: 1fr;
+        grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
```

5x 50px kan ook als repeat( 5, 50px ).
```suggestion
        grid-template-rows: 50px auto repeat( 5, 50px );
```

---

Dit heb ik nu blijkbaar als een single comment gezet ipv een review... geen idee of dit handig is zo

---

Handig als je wat verder bent en meer dan de basis beheerst, zoals jij denk ik dan wel doet haha. Maar goede tip!